tyk-stack/values.yaml

```diff
@@ -818,19 +896,22 @@ tyk-dashboard:
     # - TYK_DB_SERVEROPTIONS_CERTIFICATE_CERTFILE
     # - TYK_DB_SERVEROPTIONS_CERTIFICATE_KEYFILE
     # variables in extraEnvs object array to define your SSL cert and key files.
```

Comment:
This part can be removed as not required 🙏

Reply:
also https://github.com/TykTechnologies/tyk-charts/blob/c0bc0e52e88667cdecc05569a5b213f2d0c04d31/tyk-stack/values.yaml#L57

Reply:
thanks! removed in https://github.com/TykTechnologies/tyk-charts/pull/188/commits/35f7783f5898404799ecbbe700c4c54ccab01b74